toolkit/resources/imageconfigs/additionalfiles/iso_initrd/root/runliveinstaller

```diff
@@ -36,6 +40,13 @@ LABEL=CDROM
 
 if grep -qs $ISO_ROOT /proc/mounts; then
     echo ISO root already mounted
+# Read from /proc/cmdline to get image configs from PXE server
+elif grep -q "image-config" $CMDLINE; then
+    IMAGE_CONFIG_VALUE=$(grep -oP "(?<=--image-config=)\S+" "$CMDLINE")
+    IFS='/' read -ra CUT_DIRS <<< $IMAGE_CONFIG_VALUE
+    wget -r --no-parent -nH --cut-dirs=$((${#CUT_DIRS[@]}-3)) --directory-prefix="$CONFIG_ROOT" --reject="index.html*" "$IMAGE_CONFIG_VALUE"/
```

Comment:
Could you add a comment explaining the bash variable magic? Its not immediately clear you are getting the depth of the directory tree here (or why we need to subtract 3 from it)

Reply:
Fixed